packages/core-types/tests/integration/components/field-editors/integer-editor-test.js

```diff
@@ -0,0 +1,56 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, fillIn } from '@ember/test-helpers';
+import EmberObject from '@ember/object';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | field editors/integer editor', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    this.setProperties({
+      model: EmberObject.create({ rating: 3 }),
+      onchange() {}
+    });
+    await render(hbs`
+      {{field-editors/integer-editor
+        content=model
+        field="rating"
+        onchange=onchange
+      }}`);
+    assert.dom('input[type=number]').hasValue('3', 'number input has correct value');
+    assert.dom('input').isNotDisabled();
+  });
+
+  test('it updates the value correctly', async function(assert) {
+    this.setProperties({
+      model: EmberObject.create({ rating: 3 }),
+      onchange() {}
+    });
+    await render(hbs`
+      {{field-editors/integer-editor
+        content=model
+        field="rating"
+        onchange=onchange
+      }}`);
+    await fillIn('.field-editor > input', '5');
+    assert.dom('input[type=number]').hasValue('5', 'input is updated');
+    assert.dom('input').isNotDisabled();
+    assert.equal(this.get('model.rating'), '5', 'model attribute is updated');
```

Comment:
I think this should be:
```js
assert.strictEqual(this.get('model.rating'), 5, 'model attribute is updated');
```

as in: use `strictEqual` to make a `===` comparison, and use `5` instead of `'5'` to make sure it's actually a number

Reply:
Sure, you're right!